sycl/include/CL/sycl/accessor.hpp

```diff
@@ -816,7 +815,7 @@ class accessor :
   template <int Dims = Dimensions, typename AllocatorT,
             typename = detail::enable_if_t<
                 (Dims > 0) && (!IsPlaceH && (IsGlobalBuf || IsConstantBuf))>>
-  accessor(buffer<DataT, Dimensions, AllocatorT> &BufferRef,
+  accessor(buffer<DataT, Dims, AllocatorT> &BufferRef,
```

Comment:
Why are you changing this?

Reply:
Because calling this kind of function led to an error

Reply:
Because calling this kind of function led to an error, since this leads to instantiation of a buffer with invalid (0) parameters.